src/Settings/components/Settings.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+const SettingsScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
Normally I would say let's move this to another file such as `src/Settings/components/Settings.style.js` but this is just a placeholder page so it really doesn't matter